### ISSUE TITLE
Encrypt connection password to PG

### ISSUE THREAD

**Issue by mariec7** [pgmodeler/pgmodeler#1664]
When I opend program settings files, I found connection logon to DB in plain text. Is it going to be encrypt in future? Because it would be a security risk. Thank you for your answer.

**Comment by rkhaotix** [comment]
Yes, the same happens to the PGPASS file (which is something widely used):
https://www.postgresql.org/docs/14/libpq-pgpass.html

I understand your concern, but the settings are stored in the users local storage for a reason: they will be accessed only by the user. The local storage must be configured with the proper permissions so only the owner can access it.

Anyway, I can think of something in the future to make the password encrypted in the connection settings.
